library/src/CMakeLists.txt

```diff
@@ -76,6 +76,8 @@ if( NOT USE_CUDA )
     if( CUSTOM_ROCBLAS )
       set ( ENV{rocblas_DIR} ${CUSTOM_ROCBLAS})
       find_package( rocblas REQUIRED CONFIG NO_CMAKE_PATH )
+      target_include_directories( hipblas BEFORE PRIVATE ${CUSTOM_ROCBLAS}/include)
+
```

Comment:
Since you did the find_package what about just using ROCBLAS_INCLUDE_DIR and putting it in the list further down below.  You are making assumptions about the include that should be covered in the rocblas-config